app/controllers/services_controller.rb

```diff
@@ -17,8 +17,9 @@ def create
   end
 
   def update
+    restart = params[:restart].presence =~ /^false$/i ? false : true
```

Comment:
Can we just do

    restart = params[:restart] != 'false'

Not sure we really need to worry about case insensitivity